docs/sql-ref-ansi-compliance.md

```diff
@@ -27,35 +27,10 @@ The casting behaviours are defined as store assignment rules in the standard.
 
 When `spark.sql.storeAssignmentPolicy` is set to `ANSI`, Spark SQL complies with the ANSI store assignment rules. This is a separate configuration because its default value is `ANSI`, while the configuration `spark.sql.ansi.enabled` is disabled by default.
 
-<table class="table">
-<tr><th>Property Name</th><th>Default</th><th>Meaning</th><th>Since Version</th></tr>
-<tr>
-  <td><code>spark.sql.ansi.enabled</code></td>
-  <td>false</td>
-  <td>
-    (Experimental) When true, Spark tries to conform to the ANSI SQL specification:
-    1. Spark will throw a runtime exception if an overflow occurs in any operation on integral/decimal field.
-    2. Spark will forbid using the reserved keywords of ANSI SQL as identifiers in the SQL parser.
-  </td>
-  <td>3.0.0</td>
-</tr>
-<tr>
-  <td><code>spark.sql.storeAssignmentPolicy</code></td>
-  <td>ANSI</td>
-  <td>
-    (Experimental) When inserting a value into a column with different data type, Spark will perform type coercion.
-    Currently, we support 3 policies for the type coercion rules: ANSI, legacy and strict. With ANSI policy,
-    Spark performs the type coercion as per ANSI SQL. In practice, the behavior is mostly the same as PostgreSQL.
-    It disallows certain unreasonable type conversions such as converting string to int or double to boolean.
-    With legacy policy, Spark allows the type coercion as long as it is a valid Cast, which is very loose.
-    e.g. converting string to int or double to boolean is allowed.
-    It is also the only behavior in Spark 2.x and it is compatible with Hive.
-    With strict policy, Spark doesn't allow any possible precision loss or data truncation in type coercion,
-    e.g. converting double to int or decimal to double is not allowed.
-  </td>
-  <td>3.0.0</td>
-</tr>
-</table>
+|Property Name|Default|Meaning|Since Version|
+|--- |--- |--- |--- |
```

Comment:
Could you align the --- with the column header?
```
|Property Name|Default|Meaning|Since Version|
|-------------|-------|-------|-------------|
```

Reply:
@huaxingao Done.